<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileLocationChooser</class>
 <widget class="QDialog" name="FileLocationChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>223</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Choose location</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon.ico</normaloff>:/resources/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLineEdit" name="filePathEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please choose where Shticker Book Rewritten should save the Toontown Rewritten game files. Be sure to pick a location where your user has read, write, and execute permissions.&lt;/p&gt;&lt;p&gt;This dialog will only appear once and will save the chosen location. The location may be changed later in the settings tab of the launcher.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
